### ISSUE TITLE
Stuck with changed icon

### ISSUE THREAD

**Issue by Silther** [mollyim/mollyim-android#233]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Bug description

Signal allows you to change your icon, but if you try the same thing with molly, you cannot change it back to the original molly icon

### Steps to reproduce

1. change icon
2. try to change it back

### Molly version

v6.35.3-1

### Android version

Android 12

### Device

Samsung A52s

**Comment by valldrac** [comment]
You are right. As a workaround, just pick the first icon in the list and the original Molly icon will be back, even though it mistakenly shows the Signal icon.
